process_pr.py

```diff
@@ -483,6 +483,7 @@ def check_extra_labels(first_line, extra_labels):
 def check_type_labels(first_line, extra_labels):
     ex_labels = {}
     rem_labels = {}
+    state_labels = {}
```

Comment:
what is the purpose of this if it is not used any where :-) you can pass it as a input parameters to `check_type_labels` and then make use of it before https://github.com/cms-sw/cms-bot/blob/master/process_pr.py#L1602